tests/verify_collection.py

```diff
@@ -155,6 +157,62 @@ def collection_variables(cmr_mode, collection_concept_id, env, bearer_token):
 
     return variables
 
+def transform_grouped_dataset(nc_dataset) -> netCDF4.Dataset:
```

Comment:
this function seemed to be a copy of the l2ss function, since we have l2ss installed can you call this function directly from l2ss?